src/cpp/session/modules/SessionFind.cpp

```diff
@@ -586,7 +586,14 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
             outputStream_->flush();
             inputStream_.reset();
             outputStream_.reset();
+
+            // Unneccesary on Windows because this only sets write permissions which we
+            // already know are correct if we are writing.
+            // For efficiency, this should not be combined with the above check as flushing
+            // outputStream could change the file permissions
```

Comment:
What do you mean by the above check? Does this mean flushing the output stream should happen first? Why is it for efficiency?

Reply:
To be honest, I'm not sure. I updated the comment to be more clear.